### ISSUE TITLE
How to pass data from React Native?

### ISSUE THREAD

**Issue by alea12** [gaishimo/eas-widget-example#11]
Hello, thank you for providing this boilerplate! Related to the title, I have 2 questions:

1. How can I pass data from RN? For example, when a user updates their theme preference in the app, I want it to be reflected to the widget.
2. What is your recommended workflow to edit swift files included under `/widget/ios`? I was not able to open them with Xcode.

**Comment by kanishkaverma** [comment]
yeah i need to know this too. app groups dont seem to be working??

**Comment by alea12** [comment]
@kanishkaverma hey, I ended up using `react-native-widget-extension` to achieve what I wanted to do. I've left some comments on this thread. Hope this helps!
https://github.com/bndkt/react-native-widget-extension/pull/7
